docs/continuous-testing/react-native-testing/react-native-ui-testing-with-detox.md

```diff
@@ -95,6 +104,8 @@ If you add [Test Report Component](https://github.com/appcircleio/appcircle-test
 
 You must add this step **after** the `React Native UI Test` so that it can parse test results. Your workflow should look like the below.
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4404New-testReport.png' />
+
 :::caution Test Suites
 
 The name for Test Suites appears as undefined because the tests are not wrapped inside a describe block, which is required for the suite name to be properly displayed in the report.
```

Comment:
I think the warning mentioned here is referring to undefined which is in the first image under "Showing Test Reports". If so, can you move this warning closer to that image?

Reply:
this "caution" used in these two places. If you make a change, can you change the other one too?

Reply:
I fixed other feedbacks. But in this line, I did not understand clearly. Actually, the screen shat that mentioned in line 107 is about the looking of test results. If I move closer the screen shot, it will not be understandable. 

But, I can add a new screen shot for this.

Reply:
@boztopuz I see. When I saw the undefined text here, I thought the problem was shown in the image. But if it was done consciously, there is no problem.

<img width="1017" alt="Screenshot 2024-10-08 at 15 35 57" src="https://github.com/user-attachments/assets/075b8e51-8fa0-4a82-8a16-636a819703ca">